corebuild/integration/ILLink.Tasks/ComputeReadyToRunAssemblies.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+using System.Xml;
+using System.Xml.Linq;
+using System.Linq;
+using System.Reflection;
+using System.Collections.Generic;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using Microsoft.NET.Build.Tasks;
+using Mono.Cecil;
+
```

Comment:
Looks like some of the `using` declaration are unnecessary (e.g., `using System.Xml`). Please remove them.